DataFormats/Detectors/TRD/include/DataFormatsTRD/RawData.h

```diff
@@ -207,14 +209,15 @@ struct TrapRawTracklet { // This is a bad name as part of the tracklet data is i
   union {
     uint32_t word;
     struct {
-      uint8_t checkbit : 1; // The size of the data for this link
+      uint8_t checkbit : 1; // 
       uint16_t pid : 15;    // Particle Identity
       uint16_t slope : 6;   // Deflection angle of tracklet
       uint16_t pos : 10;    // Position of tracklet, signed 10 bits, granularity 0.02 pad widths, -10.22 to +10.22, relative to centre of pad 10
```

Comment:
I think this comment is not up to date.

Reply:
mmmm yes. pid and slope must swap around

Reply:
fixed in 68107d8d41c